eva-accession-release-automation/publish_release_to_ftp/publish_release_files_to_ftp.py

```diff
@@ -33,8 +33,9 @@
 release_file_types_to_be_checksummed = ("vcf.gz", "txt.gz", "csi")
 readme_general_info_file = "README_release_general_info.txt"
 readme_known_issues_file = "README_release_known_issues.txt"
+readme_changelog_file = "README_release_changelog.txt"
 species_name_mapping_file = "species_name_mapping.tsv"
-release_top_level_files_to_copy = (readme_known_issues_file, readme_general_info_file, species_name_mapping_file)
+release_top_level_files_to_copy = (species_name_mapping_file, readme_known_issues_file, readme_general_info_file, species_name_mapping_file)
```

Comment:
I removed the species name mapping file because I don't think it provides any useful information now that the species names are clearly provided.

Reply:
Removed? I see two entries with that variable...

Reply:
Right. Not sure how I did that but I obviously did ...